scripts/core/EXP-COMMAND-BUFFER.rst

```diff
@@ -1,14 +1,14 @@
-
 <%
     OneApi=tags['$OneApi']
     x=tags['$x']
     X=x.upper()
 %>
+
 .. _experimental-command-buffer:
 
-==============
-Command-Buffer
-==============
+================================================================================
+Command Buffer
```

Comment:
I've tried to consistently hyphenate "command-buffer" everywhere in the spec (both here & OpenCL land), so would prefer to keep this hyphenated unless there's a reason do so causes issues with the docs build

Reply:
Yep. No reason it can't be hyphenated. By default the template script with de-hyphenate, but you're free to choose any name. I'll fix this. 👍